tools/scenario-player/scenario_player/tasks/raiden_api.py

```diff
@@ -58,7 +58,12 @@ def _run(self, *args, **kwargs):
                 f'Expected {self._expected_http_status}: {resp.text}',
             )
         try:
-            return self._process_response(resp.json())
+            if resp.content == b'':
```

Comment:
Should we check here as well that the code is 204?

Reply:
It could also be another code (e.g. 201). I wouldn't want to unnecessarily restrict it to a certain code

Reply:
But I changed the comment so it's more generic